renku/api/repository.py

```diff
@@ -216,8 +216,12 @@ def is_cwl(self, path):
         """Check if the path is a valid CWL file."""
         return path.startswith(self.cwl_prefix) and path.endswith('.cwl')
 
-    def find_previous_commit(self, paths, revision='HEAD'):
-        """Return a previous commit for a given path."""
+    def find_previous_commit(self, paths, revision='HEAD', first=False):
```

Comment:
Can we rename this param to more explicit description? I propose `return_head` or `return_first`. That way its easier to reason while reading the usage of the function without going to function docs.